src/DataTable/Actions.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { MoreVert } from '../../icons';
+import {
+  Button, Dropdown, useWindowSize, Icon, breakpoints,
+} from '..';
+
+export const DROPDOWN_BUTTON_TEXT = 'More actions';
+export const SMALL_SCREEN_DROPDOWN_BUTTON_TEXT = 'Actions';
+
+// eslint-disable-next-line react/prop-types
+const CustomToggle = React.forwardRef(({ children, onClick }, ref) => (
+  <Button
+    ref={ref}
+    variant="tertiary"
+    onClick={(e) => {
+      e.preventDefault();
+      onClick(e);
+    }}
+  >
+    {children}
+  </Button>
+));
+
+const Actions = ({
+  className,
+  actionData,
+  actions,
+}) => {
+  const { width } = useWindowSize();
```

Comment:
I don't think this is something to address through this PR, but rather a thought question: should the responsive behavior of `DataTable` be based on window size, or the width of the component(s) themselves?

For example, I could imagine a potential scenario where `DataTable` is used side-by-side in a two column layout, displaying different data in each table. The window width hasn't changed but the `DataTable` width has, and the component should probably update accordingly 🤔

Reply:
🤔  Yeah, I think if we run into that use case we should update. 😄

Reply:
I agree it should be responsive to the container. There's a hook in frontend-app-learning (https://github.com/edx/frontend-app-learning/blob/master/src/generic/tabs/useIndexOfLastVisibleChild.js) That can help with this, but is potentially overkill. I agree we should defer the work for a future improvement.